src/apis/registerTool.js

```diff
@@ -9,6 +9,7 @@
  * @param {string} [properties.buttonGroup] Group of the tool button belongs to.
  * @param {string} [properties.tooltip] Tooltip of the tool button.
  * @param {function} [annotationConstructor] The constructor function for the annotation that will be created by the registered tool.
+ * @param {function} [customAnnotationCheckFxn] Function that takes in a parameter of an annotation. Returns a boolean if the specified annotation is a certain type of annotation. When this is defined, annotationConstructor can be omitted. Must define it when creating a custom annotation
```

Comment:
Is `Fxn` a widely used word? Can we use `Func` or `Function` instead?

Also, can we change this description to mention something like, `this function is used by the viewer to check if the annotation passed in is associated(created) with the registered tool`?